### PR TITLE
removing chat sounds on user status change

### PR DESCRIPTION
**Pull request by Red6Porkins**
I hate that noise on status change. I always alt-tab back to chat just to see "Userhas gone idle" DO NOT CARE

### CODE REVIEW

**Comment by wolvenhaven** [comment]
:hand: :hammer: :clock10:

File: webbage.chat.web/app/components/rooms/controllers/roomCtrl.js [hunk 1]
```diff
@@ -87,7 +89,7 @@
                                 //  2. Message isn't a connection notification
                                 //  3. Tab is not the active one (browser window DOESN'T have to be active for this)
                                 //  4. Message isn't an @reference message
-                                if (!isMyMessage && !isConnectionMessage && !isTabActive && !isReferenceMessage) {
+                                if (!isMyMessage && !isConnectionMessage && !isTabActive && !isReferenceMessage && !IsUserStatusMessage) {
```
Comment (bamblack) [inline webbage.chat.web/app/components/rooms/controllers/roomCtrl.js]: should be `!isUserStatusMessage` to follow capitalization  conventions in the rest of the project

**Comment by bamblack** [comment]
ChatHub.cs and Message.cs need to be formatted back to have same-line braces as well as the change mentioned above.

**Comment by bamblack** [comment]
Needs to be merged to dev, not master.

**Pull request closed without merging**